tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -449,126 +436,85 @@ async def test_inner_linear_normalization(
     mock.normalize.assert_not_called()
 
 
-@pytest.mark.parametrize(
-    "classifier_params, output_length",
-    [({LOSS_TYPE: "margin", RANDOM_SEED: 42, EPOCHS: 1}, LABEL_RANKING_LENGTH)],
-)
 async def test_margin_loss_is_not_normalized(
-    monkeypatch, component_builder, tmpdir, classifier_params, output_length
+    monkeypatch: MonkeyPatch, create_train_load_and_process_diet: Callable[..., Message]
 ):
-    pipeline = as_pipeline(
-        "WhitespaceTokenizer", "CountVectorsFeaturizer", "DIETClassifier"
-    )
-    assert pipeline[2]["name"] == "DIETClassifier"
-    pipeline[2].update(classifier_params)
-
     mock = Mock()
     monkeypatch.setattr(train_utils, "normalize", mock.normalize)
 
-    _config = RasaNLUModelConfig({"pipeline": pipeline})
-    (trained_model, _, persisted_path) = await rasa.nlu.train.train(
-        _config,
-        path=str(tmpdir),
-        data="data/test/many_intents.yml",
-        component_builder=component_builder,
+    parsed_message = create_train_load_and_process_diet(
+        {LOSS_TYPE: "margin", RANDOM_SEED: 42, EPOCHS: 1},
```

Comment:
Btw, created an item on the "do this later" list for being able to set the epochs to 0 (for example) here - because it would suffice to work with a randomly initialised model here.